testsuite/integration/web/src/test/java/org/jboss/as/test/integration/web/security/external/UserHandler.java

```diff
@@ -20,6 +20,7 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
         String user = exchange.getRequestHeaders().getFirst("User");
         if(user != null) {
             exchange.putAttachment(ExternalAuthenticationMechanism.EXTERNAL_PRINCIPAL, user);
+            exchange.putAttachment(HttpServerExchange.REMOTE_USER, user);
```

Comment:
This is causing a compile error. The `REMOTE_USER` cannot be found.

Reply:
Ah yeah this error is because these tests depend on my changes from undertow-io/undertow#881 to be merged.

Reply:
Sorry I missed the core-upgrade-required already on here :)